yt/frontends/swift/tests/test_outputs.py

```diff
@@ -0,0 +1,112 @@
+"""
+Swift frontend tests
+
+
+
+
+"""
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2018, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+from yt import load
+
+import numpy as np
+
+from yt.testing import requires_file
+from yt.frontends.swift.api import SwiftDataset
+from yt.testing import assert_almost_equal
+from yt.utilities.on_demand_imports import _h5py as h5py
+
+keplerian_ring = "KeplerianRing/keplerian_ring_0020.hdf5"
+EAGLE_6 = "EAGLE_6/eagle_0005.hdf5"
```

Comment:
so these tests won't actually run on appveyor or travis and I don't think we're planning on setting up jenkins for the yt-4.0 branch anytime soon. It might be a good idea to think about modifying our test setup on travis to download a minimal (small on disk, I think we only have 2 GB of storage on travis) subset of the test datasets, (in particular SPH test datasets) so the unit tests that use real data will fire on travis. I'm pretty sure there are some tests that use real datasets that are currently failing at the moment.